ROS 2 section [9292] (#194)
eProsima/Fast-RTPS-docs: docs/fastdds/use_cases/use_cases.rst

  - Disable the multicast discovery mechanism (see :ref:`use-case-disabling-multicast-discovery`).

>   - Configure a SERVER discovery mechanism (see :ref:`discovery-service-use-case`).

Reviewer: ```suggestion
  - Configure a SERVER discovery mechanism (see :ref:`discovery-server-use-case`).
```

  - Limit the blocking time of API functions to have a predictable response time (see :ref:`non-blocking-calls`).

+ :ref:`use-case-reduce-memory`.
  For use cases with memory consumption constraints, *Fast DDS* can be configured to reduce memory footprint
  to a minimum by adjusting different QoS policies.

+ :ref:`ros2`.
  Since *Fast DDS* is the default middleware implementation in the `OSRF`_ `Robot Operation System 2 (ROS 2)`_,

>   we have a whole independent section to show the use of it over ROS 2,

Reviewer: ```suggestion
  this documentation includes a whole independent section to show the use of the library in ROS 2,
```
